traffic_ops/testing/compare/README.md

```diff
@@ -104,6 +104,12 @@ optional arguments:
                         'WARN', 'ERROR', or 'CRITICAL' (default: INFO)
   -q, --quiet           Suppresses all logging output - even for critical
                         errors (default: False)
+  -s, --snapshot        Produce snapshot routes in the output (CRConfig.json,
+                        snapshot/new etc.) (default: False)
+  -C, --no-server-configs
```

Comment:
doesn't the double negative make your head tilt?

to me --server-configs default: true is easier to understand but that's just me...

Reply:
That would mean at least one flag is required to do anything, I'd like the command called with no arguments to perform a basic operation.

Reply:
No that's exactly right